nixos/modules/profiles/hardened.nix

```diff
@@ -27,6 +27,9 @@ with lib;
 
   security.forcePageTableIsolation = mkDefault true;
 
+  # This is required by podman to run containers in rootless mode.
+  security.unprivilegedUsernsClone = mkDefault (if config.virtualisation.containers.enable then true else false);
```

Comment:
Not a blocker but to me it is easier to read without the `if`, as in `mkDefault config.virtualisation.containers.enable`.

Reply:
Fixed.